docs/appendix/apis/components/sensor.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Sensor API"
+linkTitle: "Sensor"
+weight: 20
+type: "docs"
+description: "The sensor API allows you to give commands to your sensor components for getting measurements."
+icon: true
+images: ["/icons/components/sensor.svg"]
+---
+
+The sensor API allows you to give commands to your [sensor components](/components/sensor/) for getting measurements.
```

Comment:
```suggestion
The sensor API allows you to get measurements from your [sensor components](/components/sensor/).
```
or
```suggestion
The sensor API allows you to give commands for getting measurements from your [sensor components](/components/sensor/).
```
[opt] I know you're being consistent, but this feels awk and I think this suggestion makes a little more sense.